debezium-server/debezium-server-pubsub/src/main/java/io/debezium/server/pubsub/PubSubChangeConsumer.java

```diff
@@ -54,6 +64,37 @@
 
     private static final String PROP_PREFIX = "debezium.sink.pubsub.";
     private static final String PROP_PROJECT_ID = PROP_PREFIX + "project.id";
+    private static final String PROP_BATCH_DELAY_THRESHOLD = PROP_PREFIX + "batch.settings.setDelayThreshold";
```

Comment:
I don't think we need the "set" prefix"; and "settings" seems redundant, too? Also, the names should be adjusted to Debezium conventions, so this one should be `debezium.sink.pubsub.batch.delay.threshold.ms` (the `ms` suffix is common to all timeouts etc. given in milli-seconds).

Reply:
Adjusted in latest commit